### DISCUSSION TITLE
Private Labeling (Customization)

### DISCUSSION THREAD

**Discussion by mffap** [zitadel/zitadel#1916] [Announcements]
You can now customize the logo, colors, fonts of your organization's login screen in ZITADEL 🌈 

![ZITADEL Private Labeling (dark)](https://user-images.githubusercontent.com/1366906/122739588-6b704280-d283-11eb-866e-b1315758b37c.gif)

👉🏼  Checkout our docs to get started https://docs.zitadel.ch/docs/manuals/admin-policies/#private-labeling--branding

Great work, @fgerschwiler @peintnermax @livio-a

In the near future you will be able to customize texts, as well as change the design of the Console.

**Comment by fforootd** [comment]
Soon you will be able to also bring your #beautiful #branding to the #ZITADEL #console as well!

This helps #customers to recognise your #brand and also improves your #timeToMarket, because you don't need to build a #selfService interface for your #users anymore.

![console-private-labelling](https://user-images.githubusercontent.com/9879976/127190018-9f695c5c-3909-4ff0-bae2-e4c7d3322007.jpg)

**Reply by fforootd** [reply]
hopefully this is interesting for @gc-ss and @you1996 as well 😁

**Reply by gc-ss** [reply]
@fforootd Ofcourse but I think you should make this available only to paid users even if it's on your cheapest tiers.

If a company wants a branded version of the UI, they should pay you for cloud/dedicated or hire devs themselves to put a PR here.

I see that this work is done (thank you) but maybe something to consider in the future

**Reply by fforootd** [reply]
Yeah we definitely did consider this exact thought of yours 😁 

That's why this is not in the free tier of our cloud offering -> https://zitadel.ch/pricing#Privatelabeling

Dedicated customers and OSS users get it straight out of the box!

**Reply by gc-ss** [reply]
> OSS users

Ah. I actually think this idea (restricting features in the free tier) is MUCH better than removing features from OSS.

As we have seen, running the OSS in prod in HA is complicated. It will be cheaper to pay you/zitadel to do it.

You/zitadel can then run the OSS in prod in HA and continue to advertise that your dedicated offering is based off exactly the same source code as OSS (so that people can audit/review for peace of mind and security)

Any company that's not in the auth business will gladly pay you to host it for them with this guarantee.

AFAIK, no other auth provider does this! Bravo!

Brilliant!
